### PR TITLE
Week4

### PR DESCRIPTION
**Pull request by alaashhadat**

### CODE REVIEW

File: react-hyf/react-hyf-client/src/components/Week1.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const ListItem = props => {
+  const { description, deadline } = props.listInfo;
+  return (
+    <div>
+      <li>
```
Comment (Meno-101) [inline react-hyf/react-hyf-client/src/components/Week1.js]: Here it says an `li` inside of an `li`, but following the stack it's inside of a `div` then and `li`

Screen Shot 2019-07-17 at 06 28 30

File: react-hyf/react-hyf-client/src/components/Week1.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      <p>Static List</p>
+      <ul className="static-list">
+        <li class="list-group-item list-group-item-info">
+          {' '}
+          <ListItem listInfo={{ description: ' Get out of bed', deadline: 'Wed Sep 13 2017' }} />
+        </li>
+        <li class="list-group-item list-group-item-info">
+          <ListItem listInfo={{ description: 'Brush teeth', deadline: 'Thu Sep 14 2017' }} />
+        </li>
+        <li class="list-group-item list-group-item-info">
```
Comment (Meno-101) [inline react-hyf/react-hyf-client/src/components/Week1.js]: super easy fix:)
Screen Shot 2019-07-17 at 06 30 33

File: react-hyf/react-hyf-client/src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    });
+  };
+  render() {
+    return (
+      <React.Fragment>
+        <Navbar className="navbar navbar-expand-lg navbar-light bg-primary ">
+          <Container>
+            <NavbarBrand>React homework</NavbarBrand>
+            <NavbarToggler onClick={this.toggle} />
+            <Collapse isOpen={this.state.isOpen} navbar>
```
Comment (Meno-101) [inline react-hyf/react-hyf-client/src/components/Navbar.js]: These two `NavbarToggler` & `Collabse` components aren't in use as they've expected to be.

**Comment by alaashhadat** [comment]
Thank you for your time and for the feedback 🙏this is encouraging for me to keep improving my skill🙂

**Pull request closed without merging**